Tasks/AzureFileCopy/task.loc.json

```diff
@@ -11,13 +11,15 @@
   ],
```

Comment:
you need to create a new folder for a new major version of the task.

Reply:
I thought I'll do that after others have reviewed this PR. Otherwise the PR would get huge and it would become harder to review.

Reply:
done